src/arguments.rs

```diff
@@ -10,7 +10,11 @@ pub struct CommandLineArguments {
 #[derive(Debug, StructOpt)]
 pub enum Command {
     Current,
-    List,
+    #[structopt()]
+    List {
+        #[structopt(short, long, default_value = "0")]
+        number: i32,
```

Comment:
I think this is your brain on JavaScript, because this is much better as an `Option` 😂 

```suggestion
        #[structopt(short, long)]
        number: Option<usize>,
```